Fix @skipIfNoFBGEMM for types

malfet opened the pull request:
Return unmodified type from decorator if fbgemm is present.

Fix `Tried to trace <__torch__.torch.classes.rnn.CellParamsBase object at 0x55f504c56b40> but it is not part of the active trace. Modules that are called during a trace must be registered as submodules of the thing being traced` thrown from `TestPostTrainingDynamic.test_quantized_rnn`  by preserving modules in returned qRNNBase (i.e. by partially reverting https://github.com/pytorch/pytorch/pull/38134 )

malfet committed:
Fix @skipIfNoFBGEMM for types

supriyar reviewed (approved):
Thanks for the quick fix!

malfet commented:
Expect `TestPostTrainingDynamic.test_quantized_rnn` to fail, looking into potential fix

malfet committed:
Fix TestPostTrainingDynamic.test_quantized_rnn

Preserve module list in QRNNBase. This partially reverts https://github.com/pytorch/pytorch/pull/38134
